src/index.ts

```diff
@@ -152,17 +154,19 @@ export class EsbuildPlugin implements Plugin {
   async watch(): Promise<void> {
     const options = {
       ignored: this.buildOptions.watch.ignore,
-      awaitWriteFinish: true,
+      awaitWriteFinish: {
+        stabilityThreshold: 50,
```

Comment:
this it there because before it would wait 2000 ms just until it would start compiling changes

Reply:
that sounds reasonable when taking into account the size of a typical code file. But if half saved code gets compiled we'll have to higher this value a little bit (I think a couple of hundreds should still give a smooth experience but prevent any weirdness)